Rock/Reporting/DataFilter/Person/PostalCodeFilter.cs

```diff
@@ -277,12 +276,27 @@ public override Expression GetExpression( Type entityType, IService serviceInsta
                     break;
             }
 
+            IQueryable<Rock.Model.Person> qry;
+
             var groupMemberQry = groupLocationQry.Select( gl => gl.Group )
                 .Where( g => g.GroupType.Guid == familyGroupTypeGuid )
                 .SelectMany( g => g.Members );
 
-            var qry = new PersonService( ( RockContext ) serviceInstance.Context ).Queryable()
-                .Where( p => groupMemberQry.Any( xx => xx.PersonId == p.Id ) );
+            //Families which do not have locations need to be added separately
+            if ( comparisonType == ComparisonType.IsBlank || comparisonType == ComparisonType.DoesNotContain )
```

Comment:
It still not working for the _Not Equal To_ case, but I'll just fix this after I merge it in.